x-pack/plugins/infra/public/components/asset_details/tabs/overview/alerts_closed_content.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+export const AlertsClosedContent = ({ activeAlertCount }: { activeAlertCount?: number }) => {
+  const shouldRenderAlertsClosedContent = typeof activeAlertCount === 'number';
+
+  return shouldRenderAlertsClosedContent ? (
+    activeAlertCount > 0 ? (
+      <span data-test-subj="infraAssetDetailsAlertsClosedContentWithAlerts">
+        {i18n.translate('xpack.infra.assetDetails.activeAlertsContentClosedSection', {
+          defaultMessage:
+            ' {activeAlertCount} {activeAlertCount, plural, one {active alert} other {active alerts}}',
+          values: { activeAlertCount },
+        })}
+      </span>
+    ) : (
+      <span data-test-subj="infraAssetDetailsAlertsClosedContentNoAlerts">
+        {i18n.translate('xpack.infra.assetDetails.noActiveAlertsContentClosedSection', {
+          defaultMessage: 'No active alerts',
+        })}
+      </span>
+    )
+  ) : null;
```

Comment:
WDYT?
```suggestion
  const shouldRenderAlertsClosedContent = typeof activeAlertCount === 'number';
  
  if(!shouldRenderAlertsClosedContent) {
    return null;
  }

  return {
    activeAlertCount > 0 ? (
      <span data-test-subj="infraAssetDetailsAlertsClosedContentWithAlerts">
        {i18n.translate('xpack.infra.assetDetails.activeAlertsContentClosedSection', {
          defaultMessage:
            ' {activeAlertCount} {activeAlertCount, plural, one {active alert} other {active alerts}}',
          values: { activeAlertCount },
        })}
      </span>
    ) : (
      <span data-test-subj="infraAssetDetailsAlertsClosedContentNoAlerts">
        {i18n.translate('xpack.infra.assetDetails.noActiveAlertsContentClosedSection', {
          defaultMessage: 'No active alerts',
        })}
      </span>
      )
      }
```

Reply:
Perfect, thanks! Forgot to change that that after adding the second condition 👍